charts/prometheus-postgres-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "0.11.1"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 4.5.0
+version: 4.6.0
```

Comment:
@hansstandaert 
Thanks for the contribution. Since minor version bumped, do you mind to mention something that the user need to know for upgrading, e.g. characters need to escape, in README.md upgrade section?

Reply:
@zanhsieh Thank you for the quick reply. I took care of it and updated the README.md file